### DISCUSSION TITLE
WPF use webview2

### DISCUSSION THREAD

**Discussion by kopmanfc** [MicrosoftEdge/WebView2Feedback#4205] [Q&A]
"How to make the app that was built Run it on another machine and webview2 works.
Right now, I'm only working on the device that Dev uses. On other machines, when opening the app from .exe webview2 doesn't work.
What do I need to prepare to install or do?"

Thank you

**Comment by vedant297** [comment]
Hello,

I also face the same issue. Did you find any solution for this?

Thanks,
